### PR TITLE
Add change log for Web SDK 2.2.2 and 3.0.0-preview.1

### PR DESCRIPTION
**Pull request by dubiety**
Add change log for Web SDK `2.2.2` and `3.0.0-preview.1`

### CODE REVIEW

**Review by ambientlight — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,65 @@
...
+- `progressiveLoading`
+  - Enables progressive loading of map layers.
+  - Defaults to `false`.
+- `progressiveLoadingInitialLayerGroups`
+  - Specifies the layer groups to load first.
+  - Defaults to `["base"]`.
+  - Possible values are `base`, `transit`, `labels`, `buildings`, and `labels_places`.
+  - Other layer groups are deferred such that the initial layer groups can be loaded first.
+- Fix an issue that the ordering of user layers wasn't preserved after calling `map.layers.move()`.
+- Fix the initial state of `incidents` will always be true in [TrafficControlOptions][TrafficControlOptions].
```
Comment (yulinscottkang) [inline CHANGELOG.md]: This duplicates with line 40~42.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,17 +1,65 @@
...
+
+### Bug Fixes
+
+- Fix an issue that the ordering of user layers wasn't preserved after calling `map.layers.move()`.
+
+- Fix the initial state of `incidents` will always be true in [TrafficControlOptions][TrafficControlOptions].
```
Comment (yulinscottkang) [inline CHANGELOG.md]: I'm rewording this to the following:
```
- Fix the inability to disable traffic incidents in [TrafficControlOptions][TrafficControlOptions] by setting `incidents` to `false`.
```

**Review by yulinscottkang — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,17 +1,65 @@
...
+  - Specifies the layer groups to load first.
+  - Defaults to `["base"]`.
+  - Possible values are `base`, `transit`, `labels`, `buildings`, and `labels_places`.
+  - Other layer groups are deferred such that the initial layer groups can be loaded first.
+
+### Bug Fixes
+
+- Fix an issue that the ordering of user layers wasn't preserved after calling `map.layers.move()`.
+
+- Fix the inability to disable traffic incidents in [TrafficControlOptions][TrafficControlOptions] by setting `incidents` to `false`.
```
Comment (yulinscottkang) [inline CHANGELOG.md:42]: Sorry, changing it again to...
- Fix the inability to disable traffic incidents in [TrafficControlOptions][TrafficControlOptions] when `new atlas.control.TrafficControl({incidents: false})` is used.

Comment (dubiety) [reply CHANGELOG.md:42]: Will update in next PR :)

**Pull request merged**